easybuild/easyconfigs/n/NVHPC/NVHPC-20.7-GCC-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'NVHPC'
+version = '20.7'
+local_gccver = '9.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
+
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.34', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.0', '', SYSTEM),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.13', '', SYSTEM)
```

Comment:
Why not use `numactl` built with `GCC/9.3.0`?

Reply:
Paging @damianam on this one as well…

Reply:
We (JSC) need `numactl` in the `SYSTEM` toolchain due to other low-level dependencies that require it. So we took it from there as the lowest common point. But in this case, having it in `SYSTEM` or in `GCCcore` does not make a different IMO, so we can change it for the PR if you feel it fits better the overall stack.

Reply:
I think it's better not to reply on stuff built with the system compiler if it can be avoided, so please change:

```python
('numactl', '2.0.13', '', ('GCCcore', '9.3.0')),
```

Reply:
Changed.